packages/backend/src/auth.ts

```diff
@@ -38,4 +38,4 @@ function authenticateJwtTokenMiddleware(
   next()
 }
 
-function loginUser(req: Request, res: Response) {}
+export function loginUser(req: Request, res: Response) {}
```

Comment:
Why do we need it here?
We have a loginUser function in controllers/auth. Is it the same?

Reply:
Yes, it is a same. I forget to clean up.

Reply:
Fixade. 🫡